frontend/src/views/Community.vue

```diff
@@ -2,16 +2,115 @@
   <div>
     <div class="m-40 text-3xl">
       <h1>Community</h1>
+      <input
+        v-model="search"
+        type="text"
+        class="bg-gray-600"
+      >
+      <div
+        v-for="profile in results"
+        :key="profile.id"
+        class="profiles"
+      >
+        <div class="profile">
+          <img
+            class="image"
+            :src="profile.photo"
+          >
+          <div class="infoContainer">
+            <h3>{{ profile.profileName }}</h3>
+            <div class="role">
+              {{ profile.role }}
+            </div>
+            <div class="role">
+              {{ profile.userClass }}
+            </div>
+          </div><!--Closing info Container -->
+        </div><!-- Closing profile -->
+      </div><!-- Closing profiles -->
     </div>
   </div>
 </template>
 
 <script>
+import exampleProfiles from '@/example-profiles';
+import Fuse from 'fuse.js';
+
 export default {
   name: 'Community',
+  data() {
+    return {
+      fuse: null,
+      profiles: exampleProfiles,
+      search: '',
+    };
+  },
+  computed: {
+    results() {
+      if (!this.fuse || this.search == '') {
+        return this.profiles;
+      }
+      return this.fuse.search(this.search.trim()).map((result) => result.item);
+    },
+  },
+  // watch: {
+  //   search() {
+  //     if (this.search.trim() === '') {
+  //       this.result = this.profiles;
+  //     } else {
+  //       this.result = this.fuse.search(this.search.trim());
+  //     }
+  //   },
+  // },
```

Comment:
Why is this commented out?

Reply:
thank you for pointing me towards my mistake mat!